tools/train.py

```diff
@@ -120,7 +120,8 @@ def main():
         set_random_seed(args.seed, deterministic=args.deterministic)
     cfg.seed = args.seed
     meta['seed'] = args.seed
-    meta['cfg_name'] = osp.basename(args.config)
+    meta['config_name'] = osp.basename(args.config)
+    meta['work_dir'] = osp.abspath(cfg.work_dir)
```

Comment:
for privacy reasons, dir name of the work dir is good. you can use `os.path.basename(blablah.rstrip('/\\'))`